src/web/src/components/Header.vue

```diff
@@ -72,19 +82,26 @@ import { getDefaultSemester } from '@/services/AdminService';
 
 import { login } from '@/services/UserService';
 
+import SignUpComponent from '@/components/SignUp';
+
 export default {
     name: 'Header',
     props: {
       currentSemester: String
     },
+    components: {
+      SignUpForm : SignUpComponent
+    },
     data() {
       return {
         form: {
           email: '',
           password: '',
         },
         isLoggedIn: false,
+        showSingUp: false,
```

Comment:
er *`showSignUp`

Reply:
lol

Reply:
@bobmayuze can u finish this up when u get a chance, thx